coverage.html

```diff
@@ -0,0 +1,10944 @@
+
```

Comment:
Can we delete this file? It looks to just be from local development.

Reply:
I'll add to .gitignore and delete